stratum/glue/gtl/stl_util.h

```diff
@@ -54,11 +54,7 @@ inline const T* vector_as_array(const std::vector<T, Allocator>* v) {
 // '0' bytes. Typically used when code is then going to overwrite the backing
 // store of the string with known data. Uses a Google extension to ::string.
 inline void STLStringResizeUninitialized(std::string* s, size_t new_size) {
-#if __google_stl_resize_uninitialized_string
-  s->resize_uninitialized(new_size);
-#else
   s->resize(new_size);
-#endif
 }
```

Comment:
With the removal of the `#if` this whole function can go. There are no usages anyway.